src/wechaty-manager.ts

```diff
@@ -200,7 +199,7 @@ export class WechatyManager extends Manager {
   ): Promise<void> {
     log.verbose('Wechatymanager', 'onScan(%s, %s)', qrcode, status)
 
-    qrt.generate(qrcode)  // show qrcode on console
+    require('qrcode-terminal').generate(qrcode)  // show qrcode on console
```

Comment:
I remember that an error was reported when using require syntax before.

Reply:
refer to the [review](https://github.com/wechaty/matrix-appservice-wechaty/pull/100#pullrequestreview-762984536).

Reply:
At that time, the require-syntax could not pass the syntax test.